src/screens/DataApi/coverage-snapshot.json

```diff
@@ -0,0 +1,632 @@
+[
+  {
+    "name": "riskLevels.overall",
```

Comment:
I wonder if this should be a map of metric=>metric coverage rather than an array, since the order presumably isn't significant?  But if this is actually easier to consume, that's fine.  Mostly just curious.